time_slider.js

```diff
@@ -60,7 +60,7 @@ L.Control.TimeLineSlider = L.Control.extend({
       '" max="' +
       this.options.timelineDateMax.getTime() +
       '" step="any" value="' +
-      this.options.timelineDateMin.getTime() +
+      this.options.timelineDateStart.getTime() +
```

Comment:
fixing the bug of the initial timeline date, should be "date start" not "date min"